features/supports-at-rule.yml

```
@@ -0,0 +1,7 @@
+name: supports-at-rule()
```

Sorry, I didn't explain myself here very well. The `name` before was fine with me, it was only the ID/filename that needed to change.


```suggestion
name: at-rule()
```